root/edit/details/historic/AddReleaseAnnotation.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import HistoricReleaseList from '../../components/HistoricReleaseList';
+
+type AddReleaseAnnotationEditT = {
+  ...EditT,
+  +display_data: {
+    +changelog: string,
+    +html: string,
+    +releases: $ReadOnlyArray<ReleaseT>,
+    +text: string,
+  },
+};
+
+type Props = {
+  +edit: AddReleaseAnnotationEditT,
+};
+
+const AddReleaseAnnotation = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+
+  return (
+    <table className="details add-release-annotation">
+      <HistoricReleaseList
+        releases={display.releases}
+      />
+      <tr>
+        <th>{addColon(l('Text'))}</th>
+        <td>
+          {display.html
+            ? (
+              <span
+                dangerouslySetInnerHTML={{__html: display.html}}
+              />
```

Comment:
Would it be possible to use `expand2react` here, or does that not support all of the HTML that's generated?

Reply:
That's probably a good thing to check - we currently use dangerouslySetInnerHTML on every annotation-related section I think, but maybe expand2react is better. I'll try to remember to take a look tomorrow - if it works I guess we should change all of them?

Reply:
Sure. Feel free to merge this PR as-is though!